src/test/java/com/github/dockerjava/cmd/CommitCmdIT.java

```diff
@@ -44,6 +42,9 @@ public void commit() throws DockerException {
         LOG.info("Committing container: {}", container.toString());
         String imageId = dockerRule.getClient().commitCmd(container.getId()).exec();
 
+        //swarm needs some time to refelct new images
+        wait(5000);
```

Comment:
use awaitility with query to what needed?

Reply:
What do you exactly mean?

Reply:
I mean that awaitility with client query for some good to go criteria is right way. https://github.com/awaitility/awaitility